services/ui/serverless.yml

```diff
@@ -183,18 +195,17 @@ resources:
           Runtime: cloudfront-js-1.0
         Name: hsts-${self:custom.stage}
     ###############This code block enables logging on waf and sends all logs to s3.##################################
+    WaflogsUploadBucket:
```

Comment:
This bucket was previously created in the ui-waflog-s3-bucket service, which is removed as part of this PR.  That service comprised this s3 bucket, and only this s3 bucket.  It was made into its one service on sound reason and good intentions... the idea was to decouple the S3 log bucket creation, so if an account was bringing their own bucket for logs (this happens with CMS sometimes), it would be easier to adapt.  
In practice, I think removing this resource here, instead of removing the service, is easier.  And even if some argue it's more difficult, I think having one less service (especially one whose decision record is harder to explain) makes a more maintainable product.